app/assets/javascripts/discourse/tests/acceptance/opengraph-tag-updater-test.js

```diff
@@ -8,9 +8,41 @@ acceptance("Opengraph Tag Updater", function (needs) {
       return helper.response({});
     });
   });
+  needs.site({
+    anonymous_sidebar_sections: [
+      {
+        id: 111,
+        title: "Community",
+        links: [
+          {
+            id: 329,
+            name: "Everything",
+            value: "/latest",
+            icon: "layer-group",
+            external: false,
+            segment: "primary",
+          },
+          {
+            id: 331,
+            name: "Info",
+            value: "/about",
+            icon: "info-circle",
+            external: false,
+            segment: "secondary",
+          },
+        ],
+        slug: "community",
+        public: true,
+        section_type: "community",
+      },
+    ],
```

Comment:
I'm wondering if it is possible to add this to fixtures instead so that each acceptance test doesn't have to repeat this which is handful to generate.